src/components/FeedbackButton/FeedbackButton.js

```diff
@@ -0,0 +1,157 @@
+// @flow
+import React, { PureComponent } from 'react';
+import styled from 'styled-components';
+import IconBase from 'react-icons-kit';
+import { messageSquare } from 'react-icons-kit/feather/messageSquare';
+import { Spring, animated, interpolate } from 'react-spring';
+import { shell } from 'electron';
+
+import { IN_APP_FEEDBACK_URL } from '../../config/app';
+import { COLORS } from '../../constants';
+
+type Props = {
+  size: number,
+  color: string,
+  hoverColor: string,
+};
+
+type State = {
+  isHovered: boolean,
+};
+
+class FeedbackButton extends PureComponent<Props, State> {
+  static defaultProps = {
+    size: 28,
+    color: COLORS.gray[600],
+    hoverColor: COLORS.purple[500],
+  };
+
+  state = {
+    isHovered: false,
+  };
+
+  handleMouseEnter = () => {
+    this.setState({
+      isHovered: true,
+    });
+  };
+  handleMouseLeave = () => {
+    this.setState({
+      isHovered: false,
+    });
+  };
+
+  render() {
+    const { color, hoverColor, size, ...delegatedProps } = this.props;
+    const { isHovered } = this.state;
+
+    return (
+      <Spring
+        native
+        config={{
+          tension: 190,
+          friction: 40,
+        }}
+        to={{ hovered: isHovered ? 1.0 : 0 }}
+      >
+        {({ hovered }) => (

```

Comment:
@AWolf81, In order to avoid props that look like `hovered={hovered}`, @joshwcomeau suggested to not destructure the arguments of the child function, and I widely adopted this convention in my branch. I usually named the argument `interpolated` and then the prop of the child component would be `hover={interpolated.hover}`.
Other than that, it looks good to me.

Reply:
@idoberko2 thanks for reviewing.
OK, I think that's more a "style" guide thing and it looks better if not destructured. It's also more clear that the hovered value passed to the styled component is the interpolated value from React-Spring.
I'll change that.

We'll add this to our docs so it's clear that we're doing it like this. @joshwcomeau should we add it as a note to issue #210 - so we're having a reminder?